app/boards/shields/crbn/readme.md

```diff
@@ -0,0 +1 @@
+A 40% Ortho keyboard made by Polarity Works
```

Comment:
It's not essential, but you might want to include a link to your website or repository if you'd like the publicity.

Reply:
We dont have a website yet lol :D

Reply:
I guess https://keygem.store/collections/group-buys/products/group-buy-featherlight-40-kit isn't an option then?